rakudo-star/content.md

```diff
@@ -2,13 +2,13 @@
 
 Rakudo Star is a Raku (formerly known as Perl 6) distribution designed for use by early adopters of the language. It includes a virtual machine (the JVM or MoarVM), the Rakudo compiler, and a suite of modules that users may find useful. This image includes the MoarVM backend for the compiler.
 
-Project homepage: [http://rakudo.org](http://rakudo.org)
+-	Project homepage: https://rakudo.org/
+-	GitHub repository: https://github.com/rakudo/star
+-	The Dockerfile responsible: %%GITHUB-REPO%%/blob/master/Dockerfile
```

Comment:
All the `Dockerfile`s are linked above this in the generated description (pinned to the specific commit in `library/rakudo-star`, even), so I'd suggest this is probably worth just dropping completely instead.